BrowserStackLocal/BrowserStackLocal/Local.cs

```diff
@@ -92,10 +99,50 @@ private void addArgs(string key, string value)
         }
       }
     }
-    
+    public static string GetVersionString(string pVersionString)
+    {
+      string tVersion = "Unknown";
+      string[] aVersion;
+
+      if (string.IsNullOrEmpty(pVersionString)) { return tVersion; }
+      aVersion = pVersionString.Split('.');
+      if (aVersion.Length > 0) { tVersion = aVersion[0]; }
+      if (aVersion.Length > 1) { tVersion += "." + aVersion[1]; }
+      if (aVersion.Length > 2) { tVersion += "." + aVersion[2].PadLeft(4, '0'); }
+      if (aVersion.Length > 3) { tVersion += "." + aVersion[3].PadLeft(4, '0'); }
+
+      return tVersion;
+    }
+    public static Assembly GetAssemblyEmbedded(string pAssemblyDisplayName)
+    {
+       Assembly tMyAssembly = null;
+
+       if (string.IsNullOrEmpty(pAssemblyDisplayName)) { return tMyAssembly; }
+       try 
+       {
+          tMyAssembly = Assembly.Load(pAssemblyDisplayName);
+       }
+       catch (Exception ex)
+       {
+          string m = ex.Message;
```

Comment:
This exception message should be logged and/or should be sent to BrowserStack to see what caused the source to be unknown

Reply:
Done